### PR TITLE
Bevy 0.11.0-dev compatibility

### PR DESCRIPTION
**Pull request by coreh**
Hey 👋 

I needed picking for a project targeting the latest current `main` bevy branch, (v0.11.0-dev) so I've updated this crate (and also `bevy_mod_raycast`) to work under it. Here are the breaking changes I had to account for:

- Removal of base sets
- Changes to `.add_systems()`
- Y coordinate of cursor position is no longer flipped

It's likely there will be more breaking changes before the 0.11.0 release; I'll try to keep this PR up-to-date.

## To Do

- [ ] Double check all schedule assignments are indeed correct, I tried my best but there were a lot of them, I might have gotten one or two switched up by mistake. All demos (except alternate backends? see below) seem to be working correctly, though.
- [ ] Get `backend_egui` working (Might need upstream update?)
- [ ] Get `backend_rapier3d` working (Might need upstream update?)

## Related PRs

Raycasting PR: https://github.com/aevyrie/bevy_mod_raycast/pull/78

### CODE REVIEW

**Comment by aevyrie** [comment]
Damn, that was fast. Thank you!

**Comment by Vrixyz** [comment]
for the bevy_egui backend, you can check out https://github.com/mvlabat/bevy_egui/pull/188 :)

**Comment by Vrixyz** [comment]
I refreshed a bit the PR there: https://github.com/coreh/bevy_mod_picking/pull/1

File: crates/bevy_picking_input/src/touch.rs
Comment (NiklasEi) [inline crates/bevy_picking_input/src/touch.rs:1]: We should also remove the Y flipping for touch events here.

**Comment by coreh** [comment]
Closed in favor of #231

**Pull request closed without merging**